From 66a280c028f86062664322fc56be27dc875ae6df Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Mon, 2 Oct 2023 10:29:27 +0200
Subject: [PATCH] Update webos-connman-adapter.role.json.in: Add permissions to
 com.webos.service.downloadmanager

Fixes:

Oct 02 07:51:15 qemux86-64 ls-hubd[291]: [] [pmlog] ls-hubd MSG_TRUNCATED {"MSGID":"LSHUB_NO_NAME_PERMS","CAUSE":"Log message exceeded 1024 bytes","TRUNCATED_MSG":"Can not find match for 'com.webos.service.downloadmanager' in pattern queue '[\"org.webosports.wifi\", \"org.webosports.systemserv ..."}
Oct 02 07:51:15 qemux86-64 ls-hubd[291]: [] [pmlog] ls-hubd LSHUB_NO_NAME_PERMS {} Can not find match for 'com.webos.service.downloadmanager' in pattern queue '["org.webosports.wifi", "org.webosports.systemservice", "org.webosports.settingsservice", "org.webosports.service.wifi", "org.webosports.service.systemservice", "org.webosports.service.settingsservice", "org.webosports.service.pdm", "org.webosports.service.eventmonitor", "org.webosports.service.connectionmanager", "org.webosports.service.config", "org.webosports.service.activitymanager.client", "org.webosports.pdm", "org.webosports.eventmonitor", "org.webosports.connectionmanager", "org.webosports.config", "org.webosports.activitymanager.client", "org.webosinternals.wifi", "org.webosinternals.systemservice", "org.webosinternals.settingsservice", "org.webosinternals.service.wifi", "org.webosinternals.service.systemservice", "org.webosinternals.service.settingsservice", "org.webosinternals.service.pdm", "org.webosinternals.service.eventmonitor", "org.webosinternals.service.connectionmanager", "org.webosinternals.service.config", "o
Oct 02 07:51:15 qemux86-64 ls-hubd[291]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.service.downloadmanager","SRC_APP_ID":"com.webos.service.connectionmanager","EXE":"/usr/sbin/webos-connman-adapter","PID":522} "com.webos.service.connectionmanager" does not have sufficient outbound permissions to communicate with "com.webos.service.downloadmanager" (cmdline: /usr/sbin/webos-connman-adapter)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Submitted [https://github.com/webosose/webos-connman-adapter/pull/1]

 files/sysbus/webos-connman-adapter.role.json.in | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/files/sysbus/webos-connman-adapter.role.json.in b/files/sysbus/webos-connman-adapter.role.json.in
index 917c391..724c23e 100644
--- a/files/sysbus/webos-connman-adapter.role.json.in
+++ b/files/sysbus/webos-connman-adapter.role.json.in
@@ -13,7 +13,8 @@
                         "com.webos.service.pdm",
                         "com.webos.service.systemservice",
                         "com.webos.service.wifi",
-                        "com.webos.settingsservice"]
+                        "com.webos.settingsservice",
+                        "com.webos.service.downloadmanager"]
         },
         {
             "service":"com.webos.service.connectionmanager",
@@ -24,7 +25,8 @@
                         "com.webos.service.pdm",
                         "com.webos.service.systemservice",
                         "com.webos.service.wifi",
-                        "com.webos.settingsservice"]
+                        "com.webos.settingsservice",
+                        "com.webos.service.downloadmanager"]
         },
         {
             "service":"com.webos.service.wan",
@@ -35,7 +37,8 @@
                         "com.webos.service.pdm",
                         "com.webos.service.systemservice",
                         "com.webos.service.wifi",
-                        "com.webos.settingsservice"]
+                        "com.webos.settingsservice",
+                        "com.webos.service.downloadmanager"]
         }
     ]
 }
